core/services/ardupilot_manager/ArduPilotManager.py

```diff
@@ -217,12 +317,12 @@ def _reset_endpoints(self, endpoints: Set[Endpoint]) -> None:
         except Exception as error:
             logger.error(f"Error resetting endpoints: {error}")
 
-    def _update_endpoints(self) -> None:
+    def reload_endpoints(self) -> None:
         try:
             persistent_endpoints = set(filter(lambda endpoint: endpoint.persistent, self.get_endpoints()))
             self._save_endpoints_to_configuration(persistent_endpoints)
             self.settings.save(self.configuration)
-            self.restart()
+            self.restart_ardupilot()
```

Comment:
wait, why do you need to restart ardupilot here, and not only the router?

Reply:
Not needed in fact. Will change that.